submission-scripts/script-templates/magpie-hive

```diff
@@ -0,0 +1,71 @@
+#############################################################################
+##Hive configurations
+#############################################################################
+#
+# Should Hive be run
+#
+# Specify yes or no.  Defaults to no.
+#
+export HIVE_SETUP=yes
+
+# Version
+
+export HIVE_VERSION="HIVEVERSIONDEFAULT"
+
+# Path to your Hive build/binaries
+#
+# This should be accessible on all nodes in your allocation. Typically
+# this is in an NFS mount.
+#
+# Ensure the build matches the Hadoop/HDFS version this will run against.
+#
+export HIVE_HOME="HIVEDIRPREFIX/base/apache-hive-${HIVE_VERSION}-bin"
+
+# Set the port for Hive to listen on
+
+export HIVE_PORT="11112"
```

Comment:
does the user need to be able to configure this port?  Can't we just make it some standard default?  like in ```magpie-lib-defaults```?